frontend/public/components/monitoring/metrics.tsx

```diff
@@ -310,6 +303,17 @@ export const QueryTable: React.FC<QueryTableProps> = ({ index, namespace }) => {
     observe.getIn(['queryBrowser', 'lastRequestTime']),
   );
 
+  const dispatch = useDispatch();
+
+  const toggleAllSeries = React.useCallback(() => dispatch(queryBrowserToggleAllSeries(index)), [
+    dispatch,
+    index,
+  ]);
+
+  const isDisabledSeriesEmpty = useSelector(({ observe }: RootState) =>
```

Comment:
Nit: `isDisabledSeriesEmpty` is a bit like a double negative. Maybe call it `isAllSeriesEnabled` or similar?